### ISSUE TITLE
Security: Able to access to acme private key from get http API request

### ISSUE THREAD

**Issue by ltupin** [containous/traefik#3651]
### Do you want to request a *feature* or report a *bug*?
Bug

### What did you do?

Request the API simply through dashboard URL with `/api` show me the content of the acme.json letsencrypt file (the private key)
`https://monitor.company.com/api`

### What did you expect to see?

Nothing or at least **not my acme private key**.

### What did you see instead?

My private key

### Output of `traefik version`: (_What version of Traefik are you using?_)

```
Version:      v1.6.5
Codename:     tetedemoine
Go version:   go1.10.3
Built:        2018-07-10_03:54:03PM
OS/Arch:      linux/amd64
```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

```toml
logLevel = "ERROR"

defaultEntryPoints = ["http", "https"]

[entryPoints]
  [entryPoints.http]
  address = ":80"
    [entryPoints.http.redirect]
      entryPoint = "https"
  [entryPoints.https]
  address = ":443"
    [entryPoints.https.tls]
  [entryPoints.monitor]
  address = ":8080"
    [entryPoints.monitor.auth.basic]
    users = ["admin:$apr1$HKRfdjhghdfnhldgmkfjldfVaxtUrScmKc."]

[acme]
email = "email@company.com"
  storage = "acme.json"
  entryPoint = "https"
  main = "*.company.com"
  sans = ["monitor.company.com", "frontend.company.com", "backend.company.com"]
  onHostRule = true
  [acme.httpChallenge]
    entryPoint = "http"

# Enable Docker configuration backend
[docker]
  endpoint = "unix:///var/run/docker.sock"
  domain = "company.com"
  watch = true
  exposedbydefault = false

[api]
entrypoint="monitor"
```

```yml
version: "3"

networks:
  traefik-proxy:
    external: true
  internal:
    external: false

services:
  traefik-ui:
    image: traefik:tetedemoine-alpine
    container_name: monitor
    command: --docker
    labels:
      - traefik.backend=monitor
      - traefik.enable=true
      - traefik.frontend.rule=Host:monitor.company.com
      - traefik.docker.network=traefik-proxy
      - traefik.port=8080
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock:ro
      - ./traefik.toml:/traefik.toml
      - ./acme.json:/acme.json
    ports:
      - "80:80"
      - "8080:8080" # The Web UI (enabled by --api)
      - "443:443"
    networks:
      - internal
      - traefik-proxy

  frontend:
    build: ./frontend
    image: frontend:latest
    container_name: frontend
    command: gunicorn --bind 0.0.0.0 wsgi:app
    labels:
      - traefik.backend=frontend
      - traefik.frontend.rule=Host:frontend.company.com
      - traefik.docker.network=traefik-proxy
      - traefik.port=8000
    networks:
      - internal
      - traefik-proxy

  backend:
    build: ./backend
    image: backend:latest
    container_name: backend
    command: gunicorn --bind 0.0.0.0 wsgi:app
    labels:
      - traefik.backend=backend
      - traefik.frontend.rule=Host:backend.company.com
      - traefik.docker.network=traefik-proxy
      - traefik.port=8000
    networks:
      - internal
      - traefik-proxy
```

**Comment by traefiker** [comment]
Closed by #3665.

**Comment by traefiker** [comment]
Closed by #3788.
